src/modules/dummy.js

```diff
@@ -0,0 +1,32 @@
+// Dummy data
+const dummy = [{
+  name: 'Name',
+  score: 100,
+},
+{
+  name: 'Name',
+  score: 20,
+},
+{
+  name: 'Name',
+  score: 50,
+},
+{
+  name: 'Name',
+  score: 78,
+},
+{
```

Comment:
please it would be better if you remove the unused code in order to keep the file clean and pro

Reply:
ok thanks for helpful advice I will remove it